java/runtime/src/main/java/org/ray/runtime/raylet/RayletClientImpl.java

```diff
@@ -39,9 +39,9 @@
   private long client = 0;
 
   // TODO(qwang): JobId parameter can be removed once we embed jobId in driverId.
-  public RayletClientImpl(String schedulerSockName, UniqueId workerId,
+  public RayletClientImpl(String schedulerSockName, UniqueId clientId,
```

Comment:
I don't think we should revert the `clientId`->`workerId` change unless this causes problems